### ISSUE TITLE
Spatial coverage description is Iran but spatial coverage is Palestine

### ISSUE THREAD

**Issue by rybesh** [periodo/periodo-data#41]
For period http://n2t.net/ark:/99152/p03wskdwdtr the spatial coverage description is `Iran` but the spatial coverage is `Palestine`. Seems wrong?

If it is wrong, we should probably do a review of all the Pleiades spatial coverages; I know we did this automatically and there may be other weird ones.

**Comment by atomrab** [comment]
Yeah, this is a problem that I'd noticed in a couple of places in the original document Eric generated from the Pleiades dataset. I'm going to pull him and Tom in on this, because my bet is it's coming from a centroid related to a bunch of un-located places that are associated only with the bounds of a Barrington Atlas map page.

**Comment by rybesh** [comment]
Something to discuss with @paregorios:

Arguably the spatial coverage of Pleiades time periods is not expressed in the most useful way at the moment. E.g. [Classical (Greco-Roman; 550 BC-330 BC)](http://n2t.net/ark:/99152/p03wskd389m). It has no textual spatial coverage description (i.e. no general description from the Pleiades curators of the spatial region that the period covers), and it links to all of the following Wikidata places:

wikidata:Afghanistan, wikidata:Albania, wikidata:Algeria, wikidata:Armenia, wikidata:Austria, wikidata:Azerbaijan, wikidata:Bosnia and Herzegovina, wikidata:Bulgaria, wikidata:Croatia, wikidata:Cyprus, wikidata:Czech Republic, wikidata:Egypt, wikidata:France, wikidata:Georgia, wikidata:Germany, wikidata:Gibraltar, wikidata:Greece, wikidata:Hungary, wikidata:India, wikidata:Iran, wikidata:Iraq, wikidata:Israel, wikidata:Italy, wikidata:Jordan, wikidata:Kenya, wikidata:Kosovo, wikidata:Kuwait, wikidata:Kyrgyzstan, wikidata:Lebanon, wikidata:Libya, wikidata:Macedonia, wikidata:Malta, wikidata:Moldova, wikidata:Monaco, wikidata:Montenegro, wikidata:Morocco, wikidata:Oman, wikidata:Pakistan, wikidata:Palestine, wikidata:Portugal, wikidata:Romania, wikidata:Russia, wikidata:Saudi Arabia, wikidata:Serbia, wikidata:Slovenia, wikidata:Somalia, wikidata:Spain, wikidata:Sudan, wikidata:Switzerland, wikidata:Syria, wikidata:Tajikistan, wikidata:Tunisia, wikidata:Turkey, wikidata:Turkmenistan, wikidata:Ukraine, wikidata:United Kingdom, wikidata:Uzbekistan, wikidata:Yemen

Presumably, that long list of countries could be replaced with some broader modern regions, or with historical regions. I'm not sure what makes most sense for people using Pleiades + PeriodO data, but the status quo seems sub-optimal.

**Comment by atomrab** [comment]
The way we derived the spatial coverage in Pleiades was unique, and I think still useful: rather than trying to take a statement of spatial coverage from the label or asking @paregorios to make a list, we took the entire dataset and inferred spatial coverage from the sum of the locations of all of the places tagged with a given period label. The spatial coverage takes the form of modern nations because we were still using DBpedia at that point. 

There's some noise in there, either from the very broad application of those period labels in the Barrington Atlas or from mistakes in Pleiades, but this seems to me a valuable proof of concept for the inferential establishment of spatial coverage for a period from place locations in an existing gazetteer (as opposed to the top-down designation, which presents its own challenges). I'm not averse to consolidating, but can we do that while maintaining the bottom-up calculation from site locations in this case?
